packages/react-frontend/src/NewComment.js

```diff
@@ -15,16 +15,15 @@ function NewComment(props) {
   }
 
   function handleSubmit(event) {
+    event.preventDefault();
```

Comment:
What does this do?

Reply:
This prevents the form from having default values posted. It also somehow fixes the issue where on firefox the http post fails with status NS_BINDING_ABORTED.